### DISCUSSION TITLE
remove #review tags or stop reviewing a note

### DISCUSSION THREAD

**Discussion by jk8104** [st3v3nmw/obsidian-spaced-repetition#375] [General]
Is there and easy way to stop reviewing a note? I use the hotkeys to do my "hard, good, easy" quick reviews, but it would be nice to have an option to "stop reviewing" as well. Is the only way to remove the #review tag and the YAML frontmatter? Thanks and thank for the great plugin!!!

**Comment by st3v3nmw** [comment]
Hey, currently no. You could open a feature request.
